inference-engine/tests/functional/plugin/shared/include/behavior/infer_request/multithreading.hpp

```diff
@@ -18,9 +31,13 @@ TEST_P(InferRequestMultithreadingTests, canRun3SyncRequestsConsistentlyFromThrea
     auto req3 = execNet.CreateInferRequest();
 
 
-    auto f1 = std::async(std::launch::async, [&] { req1.Infer(); });
-    auto f2 = std::async(std::launch::async, [&] { req2.Infer(); });
-    auto f3 = std::async(std::launch::async, [&] { req3.Infer(); });
+    auto f1 = async([&] { req1.Infer(); });
```

Comment:
If this is true about std::async, does it mean that users also should not infer asynchronously using `std::async`?

Reply:
Can not answer. The condition of the hang is still unclear.